pylib/cqlshlib/cql3handling.py

```diff
@@ -296,6 +296,10 @@ def dequote_value(cqlword):
                             | <alterRoleStatement>
                             | <dropRoleStatement>
                             | <listRolesStatement>
+                            | <createSlaStatement>
+                            | <alterSlaStatement>
+                            | <dropSlaStatement>
+                            | <listSlaStatement>
```

Comment:
there are `detach` and `attach` statements

Reply:
added those